core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
-        <span className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">right-0</span>
-        <span className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">bottom-0</span>
-        <span className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">left-0</span>
+
+      {/* Border color examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Colors
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="border border-primary bg-white p-8 d-inline-block mr-6">
+            border-primary
+          </Text>
+          <Text appearance="default" className="border border-secondary bg-white p-8 d-inline-block mr-6">
+            border-secondary
+          </Text>
+          <Text appearance="default" className="border border-success bg-white p-8 d-inline-block mr-6">
+            border-success
+          </Text>
+          <Text appearance="default" className="border border-warning bg-white p-8 d-inline-block mr-6">
+            border-warning
+          </Text>
+          <Text appearance="default" className="border border-alert bg-white p-8 d-inline-block mr-6">
+            border-alert
+          </Text>
+          <Text appearance="default" className="border border-accent1 bg-white p-8 d-inline-block mr-6">
+            border-accent1
+          </Text>
+          <Text appearance="default" className="border border-accent2 bg-white p-8 d-inline-block mr-6">
+            border-accent2
+          </Text>
+          <Text appearance="default" className="border border-accent3 bg-white p-8 d-inline-block mr-6">
+            border-accent3
+          </Text>
+          <Text appearance="default" className="border border-accent4 bg-white p-8 d-inline-block">
+            border-accent4
+          </Text>
+        </div>
+        <div className="mb-5 bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {
+                '<Text appearance="default" className="border border-primary bg-white p-8 d-inline-block mr-6">border-primary</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-secondary bg-white p-8 d-inline-block mr-6">border-secondary</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-success bg-white p-8 d-inline-block mr-6">border-success</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-alert bg-white p-8 d-inline-block mr-6">border-alert</Text>'
+              }
+            </code>
+          </pre>
```

Comment:
Instead of using multiple <code> elements we can simply write all the text inside single <code>